frontend/src/app/features/invite-user-modal/summary/summary.component.sass

```diff
@@ -1,2 +1,6 @@
-.op-ium-summary-message
-  white-space: pre-wrap 
+.op-ium-summary
+  &-message
+    white-space: pre-wrap 
+
+  &-details 
```

Comment:
This creates two blocks in the same sass file. I think it makes more sense to use `--details` and `--message` and adding the `op-ium-summary` block class to the wrapping form.